topdown/strings.go

```diff
@@ -241,6 +241,71 @@ func builtinTrim(a, b ast.Value) (ast.Value, error) {
 	return ast.String(strings.Trim(string(s), string(c))), nil
 }
 
+func builtinTrimLeft(a, b ast.Value) (ast.Value, error) {
```

Comment:
Can you add some basic unit tests for these? Just thinking like basic smoke or regression tests should do since these don't have a lot of complexity in them.

Reply:
865da93 addresses this.